packages/patch/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@graphql-inspector/patch",
```

Comment:
I think you should use `0.0.0` as version, and add a changeset, this will ensure to have the first release as part of the Releases tab.